README.md

```diff
@@ -1,3 +1,12 @@
 # rolldown
 
 [![Travis build status](https://travis-ci.com/yihui/rolldown.svg?branch=master)](https://travis-ci.com/yihui/rolldown)
+![](https://img.shields.io/badge/lifecycle-experimental-orange.svg)
```

Comment:
Please try your best to add the alt text to images whenever it is possible (consider visually impaired people): https://yihui.name/cn/2017/04/img-alt/

Reply:
I will take this by heart.